api/database/seeders/ClassificationRoleSeeder.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+
+class ClassificationRoleSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+      $roles = [
+            [
+                'key' => 'technician_it01 ',
+                'name' => [
+                    'en' => 'Technician ',
```

Comment:
Try asking Gray for the french translations of these roles. It would be good to get them into this seeder.